harness/harness-core-ui
# feat: [CI-4470]: flaky buildInfraSpecificationsTest

##### Summary:



##### Jira Links:



##### Screenshots:

Flaky test with log:

FAIL src/modules/75-ci/components/PipelineStudio/BuildInfraSpecifications/__test__/BuildInfraSpecifications.test.tsx
  ● Test suite failed to run

    Call retries were exceeded

      at ChildProcessWorker.initialize (node_modules/jest-worker/build/workers/ChildProcessWorker.js:193:21)
      at ChildProcess.emit (node:events:390:28)
      at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)

https://app.harness.io/ng/#/account/vpCkHKsDSxK9_KYfjCTMKA/ci/orgs/default/projects/NextGenUI/pipelines/Jest/executions/x4Mf7_OIQhOMuP_e-uUK9w/pipeline

Other solutions suggest upgrading the node version on the build server


You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`
